scantools/scanners/navvis/navvis.py

```diff
@@ -116,6 +116,25 @@ def load_frames(self):
         """
         # get valid frame_ids parsing image folder
         valid_ids = sorted(self._input_image_path.glob("*-cam0.jpg"))
+
+        # If no jpg images were found, try converting dng images to jpg.
+        # This occurs when you don't have access to the datasets_proc/ folder,
+        # but only to the .nvd (from Navvis cloud) + datasets_rec/ folder.
+        if len(valid_ids) == 0:
+            dng_images = sorted(self._input_image_path.glob(f"*.dng"))
```

Comment:
here we look for *.dng, but below for -cam0.jpg. Does it mean, the dng files are actually called *-cam0.dng?

Reply:
we want to process all the images, so camX. The cam0 was to get the valid IDs. We used to have a file with the valid IDs, but navvis removed it, so we need to do this hack with the cam0 to find out the valid frame IDs.